example/session/src/main/java/org/apache/iotdb/DataMigrationExample.java

```diff
@@ -174,13 +214,13 @@ public Void call() {
 
       } catch (Exception e) {
         System.out.println(
-            "Loading the " + i + "-th timeseries: " + series + " failed " + e.getMessage());
+            "Loading the " + i + "-th devices: " + device + " failed " + e.getMessage());
         return null;
       } finally {
         readerPool.closeResultSet(dataSet);
       }
 
-      System.out.println("Loading the " + i + "-th timeseries: " + series + " success");
+      System.out.println("Loading the " + i + "-th devices: " + device + " success");
```

Comment:
The number of loaded devices is not correct, right? Is there any way to print the correct number?